inst/examples/installBioc.r

```diff
@@ -24,20 +34,93 @@ package 'BiocManger' which has be installed.
 
 Examples:
   installBioc.r -l /tmp/lib S4Vectors               # install into given library
+  installBioc.r --update Biobase                    # install package and update older packages
+  installBioc.r --deps NA --error --skipinstalled   # install package without suggested dependencies, throw an error on installation failure and skip packages that are already present
 
 installBioC.r is part of littler which brings 'r' to the command-line.
 See http://dirk.eddelbuettel.com/code/littler.html for more information.\n")
     q("no")
 }
 
 if (!requireNamespace("BiocManager", quietly=TRUE)) {
-    stop("Please install 'BiocManager' first, for example via 'install.r BiocManager'.", call.=FALSE)
+    stop("Please install 'BiocManager' first, for example via 'install2.r BiocManager'.", call.=FALSE)
```

Comment:
Nit:  Why the change on that line?  `install.r BiocManager` also works.

(Minutes later:  Ah well if this script models itself after `install2,r`, maybe?  Oh well -- both will get BiocManager installed...)

Reply:
I revert it back to `install.r` simply because that one is older + simpler + (maybe?) more widely know.  But it is a non-issue.